### ISSUE TITLE
Fails when called from worker

### ISSUE THREAD

**Issue by jackkleeman** [HenrikJoreteg/webrtcsupport#29]
This function seems to expect document and window globals to exist, giving a runtime error when called in a web worker. As far as I know, webrtc support does not exist within a worker, so a simple fix is to check if document and window exist, and if not then return false for all relevant booleans.

**Comment by daviddias** [comment]
Yeah, this is the last task to get https://github.com/ipfs/js-ipfs/issues/725 done  too :)
